tests/org.eclipse.passage.lic.net.tests/src/org/eclipse/passage/lic/net/tests/io/KeyKeeperWithOppositeLineDelimiterFactory.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IILS mbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IILS mbH (Hannes Wellmann) - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.net.tests.io;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+
+import org.eclipse.passage.lic.api.LicensingException;
+import org.eclipse.passage.lic.api.io.KeyKeeper;
+import org.eclipse.passage.lic.base.io.FileKeyKeeper;
+
+final class KeyKeeperWithOppositeLineDelimiterFactory extends TestKeyKeeper {
+
+	@Override
+	public final KeyKeeper get() throws LicensingException, IOException {
```

Comment:
all methods in `final` class are practically `final`. 
Here the modificator is excessive.